lib/internal/modules/cjs/loader.js

```diff
@@ -733,6 +748,12 @@ Module._extensions['.json'] = function(module, filename) {
 
 // Native extension for .node
 Module._extensions['.node'] = function(module, filename) {
+  var content = fs.readFileSync(filename);
```

Comment:
these two lines should definitely be inside the condition

Reply:
fixed